.gitignore

```diff
@@ -19,3 +19,8 @@ runs/
 
 # logs should also be ignored
 logs/
+
```

Comment:
I don't think we need these... the preferred layout is to check out Anserini in `../anserini`.